gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleTooling.java

```diff
@@ -63,9 +63,20 @@
             final String initScriptTemplate =
                 CoreUtil.readStreamToString( GradleTooling.class.getResourceAsStream( "init.gradle" ) );
 
-            String initScriptContents = initScriptTemplate.replaceFirst(
-                "%model%", modelBundle.getAbsolutePath().replaceAll( "\\\\", "/" ) ).replaceFirst(
-                    "%plugin%", pluginBundle.getAbsolutePath().replaceAll( "\\\\", "/" ) );
+            String modelBundlePath = modelBundle.getAbsolutePath().replaceAll( "\\\\", "/" );
+            String pludinBundlePath = pluginBundle.getAbsolutePath().replaceAll( "\\\\", "/" );
+
+            // IDE-2296
+            if( CoreUtil.isMac() )
+            {
+                modelBundlePath =
+                    modelBundlePath.replace( "Eclipse.app/Contents/MacOS", "Eclipse.app/Contents/Eclipse" );
```

Comment:
There are other Eclipse distributions that don't use "Eclipse.app" like DeveloperStudio.app etc, so can we improve this to not including Eclipse.app